cogs/numb.py

```diff
@@ -0,0 +1,63 @@
+'''This is a guess the number game'''
+import random
+import asyncio
+from discord.ext import commands
+
+
+GuessesTaken = 0
+
+class Guess:
```

Comment:
This conflicts with #74.  Please choose a different name, or abandon your previous PR.